src/services/persisters/s3Persister.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+const AWS = require('aws-sdk');
+
+const bucketName = "mirai-app";
+const MongoDbPersister = require('./persisters/mongoDbPersister');
+
+
+class s3Persister {
+    constructor(root) {
+        this.mongo = new MongoDbPersister(this);
+    }
+
+    get(key, body) {
+        const params = {
+            Bucket: bucketName,
+            Key: key,
+            Body: body
+        };
+        s3.download(params).then(data => { }).catch(err => { });
+    }
+
+    save(key, body) {
+        const params = {
+            Bucket: bucketName,
+            Key: key,
+            Body: body
+        };
+        s3.upload(params).then(data => { }).catch(err => { });
```

Comment:
What is "`s3`"? `s3.upload` isn't valid because 1. `upload()` isn't a function anywhere in the `aws-sdk` and 2. `s3` isn't instantiated.
Make sure you're following the tutorial properly.

Reply:
At the moment, this doesn't do anything except throw.